actionpack/lib/action_dispatch.rb

```diff
@@ -75,7 +75,10 @@ class MissingController < NameError
 
   autoload :Journey
   autoload :MiddlewareStack, "action_dispatch/middleware/stack"
-  autoload :Routing
+
+  eager_autoload do
```

Comment:
Do we need this given we call `eager_load!` on `eager_load!`

Reply:
Yeah, I guess not.